File: HotelManager-Frontend/src/components/roomType/RoomType.js
const RoomType = ({ roomType }) => {
    const { id, nameRoomType, description } = roomType;

    return (
        <div className="room-type" key={id}>
            <h2>Name of room type:{nameRoomType}</h2>
            <p>Description: {description}</p>
        </div>
    ); 
}; 

File: HotelManager/HotelManager.Model/RoomType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HotelManager.Model
{
    public class RoomType
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public Guid CreatedBy { get; set; }
        public Guid UpdatedBy { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime DateUpdated { get; set; }
        public bool IsActive { get; set; }
        public RoomType() { }
    }
}
